packages/web-wallet/package.json

```diff
@@ -117,31 +118,31 @@
   },
   "devDependencies": {
     "@jest/types": "^29.6.3",
-    "@mui/styles": "^5.16.1",
+    "@mui/styles": "^5.16.7",
     "@sphereon/pex": "5.0.0-unstable.14",
-    "@sphereon/pex-models": "^2.2.4",
-    "@tanstack/query-core": "^5.51.1",
-    "@testing-library/jest-dom": "^6.4.6",
+    "@sphereon/pex-models": "^2.3.1",
+    "@tanstack/query-core": "^5.59.16",
+    "@testing-library/jest-dom": "^6.6.2",
     "@testing-library/react": "^14.3.1",
     "@testing-library/user-event": "^14.5.2",
     "@types/cookie": "^0.5.2",
     "@types/debug": "^4.1.12",
-    "@types/jest": "^29.5.12",
-    "@types/node": "^18.19.39",
-    "@types/react": "^18.3.3",
-    "@types/react-dom": "^18.3.0",
+    "@types/jest": "^29.5.14",
+    "@types/node": "^18.19.59",
+    "@types/react": "^18.3.12",
+    "@types/react-dom": "^18.3.1",
     "@types/sha.js": "^2.4.4",
     "@types/uuid": "^9.0.8",
     "@typescript-eslint/parser": "5.62.0",
     "cross-env": "^7.0.3",
     "css-loader": "^6.11.0",
-    "husky": "^9.0.11",
+    "husky": "^9.1.6",
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
     "prettier": "^3.3.3",
     "sass-loader": "^14.2.1",
     "style-loader": "^3.3.4",
-    "ts-jest": "^29.2.2",
+    "ts-jest": "^29.2.5",
     "ts-node": "^10.9.2",
     "typescript": "4.9.5",
```

Comment:
still using 4, main is using 5

Reply:
I do not understand the statement. we do not use the main branch

Reply:
fixed